tools/update_version.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+
+# Copyright (c) 2023 Elektrobit Automotive GmbH
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License, Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+set -e
+
+script_dir=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+base_dir="$script_dir/.."
+workspace_config="$base_dir/Cargo.toml"
+
+usage() {
+    echo "Usage: $0 [--release] VERSION"
+    echo "Update Ankaios files to VERSION."
+    echo "  --release Official release with assets for download."
+    exit 1
+}
+
+log_update() {
+    echo "Updating $(realpath -e --relative-base="$(pwd)" "$1")"
+}
+
+# Initialize variables
+release=0
+version=""
+
+# Parse arguments
+while [[ "$#" -gt 0 ]]; do
+    case $1 in
+        --release) release=1; shift ;;
+        -h|--help) usage ;;
+        *)
+            if [[ -z "$version" ]]; then
+                version="$1"
+            else
+                echo "Error: Unknown parameter passed: $1"
+                usage
+            fi
+            shift
+            ;;
+    esac
+done
+
+# Check if VERSION is set
+if [[ -z "$version" ]]; then
+    echo "Error: VERSION is a mandatory argument."
+    usage
+fi
+
+# Extract all packages from the workspace file
+packages=$(awk '/members *= *\[/{flag=1; next} /\]/{flag=0} flag {gsub(/[" ,]/, ""); print}' "$workspace_config")
```

Comment:
To be more robust against newlines between package names, I suggest the following small change:
```suggestion
packages=$(awk '/members *= *\[/{flag=1; next} /\]/{flag=0} flag && NF {gsub(/[" ,]/, ""); print}' "$workspace_config")
```

Reply:
I have tested the existing script and it already works with empty lines between the members. The awk statement will then just output an empty item which will be ignored by the line `for pkg in $packages; do`.
For that reason I think we not need to include this change.

Reply:
Ok, I think we can keep it.